src/Altinn.Notifications.Core/Integrations/IRegisterClient.cs

```diff
@@ -27,5 +27,5 @@ public interface IRegisterClient
     /// A task that represents the asynchronous operation. 
     /// The task result contains a list of <see cref="PartyDetails"/> representing the details of the specified individuals and organizations.
     /// </returns>
-    Task<List<PartyDetails>> GetPartyDetails(List<string> organizationNumbers, List<string> socialSecurityNumbers);
+    Task<List<PartyDetails>> GetPartyDetails(List<string>? organizationNumbers = null, List<string>? socialSecurityNumbers = null);
```

Comment:
Adding null as default value was probably unnecessary.

My opinion on default values are that we should avoid them. Use them only to retain backwards compatibility when adding a parameter. That way you don't need to update all call points. Especially useful in a library. When the call points are few it is better to update the call points.